Editor/Utility/CinemachineSceneTools.cs

```diff
@@ -0,0 +1,381 @@
+                            lens.Aspect).ToString("F1") + ")");
+                }
+                else
+                {
+                    DrawLabel(labelPos, "Vertical FOV (" + 
+                        lens.FieldOfView.ToString("F1") + ")");
+                }
+            }
+                
+            if (GUIUtility.hotControl == fovHandleId) 
+                CinemachineBrain.SoloCamera = vcam;
+        }
+
+        public static void NearFarClipHandle(CinemachineVirtualCameraBase vcam, ref LensSettings lens)
+        {
+            var camPos = vcam.State.FinalPosition;
+            var camRot = vcam.State.FinalOrientation;
+            var camForward = camRot * Vector3.forward;
+            var nearClipPos = camPos + camForward * lens.NearClipPlane;
+            var farClipPos = camPos + camForward * lens.FarClipPlane;
+            
+            EditorGUI.BeginChangeCheck();
+            var ncHandleId = GUIUtility.GetControlID(FocusType.Passive);
+            var newNearClipPos = Handles.Slider(ncHandleId, nearClipPos, camForward, 
+                HandleUtility.GetHandleSize(nearClipPos) / 10f, Handles.CubeHandleCap, 0.5f); // division by 10, because this makes it roughly the same size as the default handles
+            var fcHandleId = GUIUtility.GetControlID(FocusType.Passive);
+            var newFarClipPos = Handles.Slider(fcHandleId, farClipPos, camForward, 
+                HandleUtility.GetHandleSize(farClipPos) / 10f, Handles.CubeHandleCap, 0.5f); // division by 10, because this makes it roughly the same size as the default handles
+            if (EditorGUI.EndChangeCheck())
+            {
+                Undo.RecordObject(vcam, "Changed clip plane using handle in scene view.");
+                
+                lens.NearClipPlane += 
+                    SliderHandleDelta(newNearClipPos, nearClipPos, camForward);
+                lens.FarClipPlane += 
+                    SliderHandleDelta(newFarClipPos, farClipPos, camForward);
+                lens.Validate();
+                
+                InspectorUtility.RepaintGameView();
+            }
+
+            if (GUIUtility.hotControl == ncHandleId || HandleUtility.nearestControl == ncHandleId)
+            {
+                DrawLabel(nearClipPos,
+                    "Near Clip Plane (" + lens.NearClipPlane.ToString("F1") + ")");
+            }
+            if (GUIUtility.hotControl == fcHandleId || HandleUtility.nearestControl == fcHandleId)
+            {
+                DrawLabel(farClipPos, 
+                    "Far Clip Plane (" + lens.FarClipPlane.ToString("F1") + ")");
+            }
+
+            
+            if (GUIUtility.hotControl == ncHandleId || GUIUtility.hotControl == fcHandleId) 
+                CinemachineBrain.SoloCamera = vcam;
+        }
+
+        public static void TrackedObjectOffsetTool(
+            CinemachineComponentBase cmComponent, ref Vector3 trackedObjectOffset, string tag)
```

Comment:
You don't need the "tag" param.  You can get the tag by looking at cmComponent.Stage